aiven/client/cli.py

```diff
@@ -2675,6 +2675,51 @@ def credits__claim(self):
         if self.args.json:
             self.print_response(result, json=True)
 
+    @arg.json
+    @arg.project
+    @arg("--service", help="Related service name")
+    @arg("--severity", required=True, choices=["low", "high", "critical"], help="Ticket severity")
+    @arg("--title", required=True, help="Short description")
+    @arg("--description", required=True, help="Longer description")
+    def ticket_create(self):
```

Comment:
Even though a single underscore works, for consistency it is better to have double underscore for the first split as well (i.e., `ticket__create` and `ticket__list`). This is a very small thing, but makes searching for specific methods nicer, as you always know any breaks are with double underscore.